manifests/j/joncampbell123/DOSBox-X/dosbox-x-v0.83.18/joncampbell123.DOSBox-X.locale.en-US.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.7-1-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: joncampbell123.DOSBox-X
+PackageVersion: dosbox-x-v0.83.18
```

Comment:
```suggestion
PackageVersion: 0.83.18
```
Well - looks like only 1.1 can save us from upgrade loops. For now, just remove the `dosbox-x-v` bit from automation.
![image](https://user-images.githubusercontent.com/11600822/135753479-0eaa608b-2a3c-4abd-b925-1ac603355384.png)
![image](https://user-images.githubusercontent.com/11600822/135753531-bc27a351-3e0c-4f10-8ba5-59040f29b671.png)